product_docs/docs/pgd/5/cli/index.mdx

```diff
@@ -13,15 +13,15 @@ directoryDefaults:
   description: "The PGD Command Line Interface (CLI) is a tool to manage your EDB Postgres Distributed cluster"
 ---
 
-The EDB Postgres Distributed Command Line Interface (PGD CLI) is a tool for managing your EDB Postgres Distributed cluster. It allows you to run commands against EDB Postgres Distributed clusters. It may be installed automatically on systems within a TPA-deployed PGD cluster or it can be installed manually on systems that can connect to any PGD cluster, including BigAnimal Distributed High Availability PGD clusters or PGD clusters deployed using the EDB PGD for Kubernetes operator.
+The EDB Postgres Distributed Command Line Interface (PGD CLI) is a tool for managing your EDB Postgres Distributed cluster. It allows you to run commands against EDB Postgres Distributed clusters. It can be installed automatically on systems in a TPA-deployed PGD cluster. Or it can be installed manually on systems that can connect to any PGD cluster, including BigAnimal distributed high-availability PGD clusters or PGD clusters deployed using the EDB PGD for Kubernetes operator.
```

Comment:
It is installed automatically on each node in a PGD cluster deployed using TPA.  You may want to install it manually to connect to a TPA-deployed cluster remotely.  

The PGD CLI needs to be installed manually for use with PGD clusters deployed to EDB BigAnimal or with the EDB Postgres Distributed for Kubernetes operator.

Reply:
we probably need to agree to how we want to handle the BA references but there's a Cluster Type in the UI and the option is named Distributed High Availability so I don't think chaning them all to "distributed high-avialability" makes sense